melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/eux/EuxService.java

```diff
@@ -95,6 +100,13 @@ public void oppdaterSed(String rinaSaksnummer, String dokumentId, SED sed) {
 
     public void opprettOgSendSed(SED sed, String rinaSaksnummer) {
         String sedId = euxConsumer.opprettSed(rinaSaksnummer, sed);
+        if (unleash.isEnabled("melosys.eessi.handlingssjekk_sed")) {
+            if (!sedHandlingErMulig(rinaSaksnummer, sedId, SedHandlinger.Send.hentHandling())) {
+                log.warn("Ugyldig handling {} for Sed {}", SedHandlinger.Send.hentHandling(), sedId);
+                throw new IntegrationException("Kan ikke sende SED, ugyldig handling i Rina");
```

Comment:
Tenkte å bruke `ValidationException` her i stedet.

Reply:
Nja, det er jo også integrasjonen med eux-rina som er ulovlig. Ser også at ValidationException må håndteres manuelt, så da må jeg refaktorere alle kall mot `opprettOgSendSed`

Reply:
Jeg ser det og `ValidationException` er kanskje ikke egnet. Imidlertid er `IntegrationException` så langt bruk for litt mer brå feil, og mappes til `INTERNAL_SERVER_ERROR` i `RestExceptionHandler`. Så `IntegrationException` passer heller ikke. Skal vi introdusere en ny?